index.js

```diff
@@ -30,9 +30,10 @@ var absolution = module.exports = function(input, base, options) {
       _.forEach(attribs, function(value, a) {
         result += ' ' + a;
         if (value.length) {
+          const quote = value.startsWith('{"') ? `'` : `"`;
```

Comment:
```
input: `<div data-test1='{"foo":"bar"}'>Test</div>`
attribs: { 'data-test': '{"foo":"bar"}' }
value: `{"foo":"bar"}`
```

I only found the `startWith('{"')` solution, but there is probably a better way, only I can't find it...